code/game/objects/structures/kitchen_spike.dm

```diff
@@ -66,10 +83,14 @@
 				usr << "You remove the last piece of meat from the monkey!"
 				src.icon_state = "spike"
 				src.occupied = 0
-		else if(src.meattype == 2)
-			if(src.meat > 1)
+		else if(src.meattype == MEATTYPE_ALIEN && src.skintype == SKINTYPE_ALIEN)
+			if(src.skin == 1)

```

Comment:
a creature literally only has 1 skin, and every usage of the variable is either 0 or 1.